src/dmd/utils.d

```diff
@@ -210,6 +140,64 @@ void escapePath(OutBuffer* buf, const(char)* fname)
     }
 }
 
+/**
+ * Takes a path, and make it compatible with Makefile format.
+ *
+ * That is, turns
+ *  - '\' into '/'
+ *  - '$' into "$$"
+ *  - ' ' into "\ "
+ *  - '%' into "\%"
+ *  - '#' into "\#"
+ *  - ':' into "\:"
+ *
+ * Params:
+ *   buf = Buffer to write the escaped path to
+ *   fname = Path to escape
+ */
+void escapeMakePath(OutBuffer *buf, const(char)* fname)
+{
+    while(*fname)
+    {
+        switch (*fname)
+        {
+            case '\\':
+                buf.writeByte('/');
```

Comment:
I'm not sure this is correct.

Reply:
This is not escape, but path normalization between windows and unix.
Probably not strictly necessary, but help to disambiguate the other escapes.

Reply:
Also makes testing more predictable on Windows